app/controllers/ops_controller.rb

```diff
@@ -790,7 +790,7 @@ def handle_bottom_cell(nodetype, presenter, locals)
       if @pages
         presenter.hide(:form_buttons_div)
       elsif @in_a_form
-        if nodetype == "log_depot_edit"
+        if nodetype == "log_depot_edit" || "ze"
```

Comment:
Pretty sure this should be `if nodetype == "log_depot_edit" || nodetype == "ze"`